edx_event_bus_kafka/internal/producer.py

```diff
@@ -51,7 +51,7 @@ def record_producing_error(error, context):
         raise Exception(error)
     except BaseException:
         record_exception()
-        logger.exception(f"Error producing event to event bus. {error=!s} {context!r}")
+        logger.exception(f"Error delivering message to Kafka event bus. {error=!s} {context!r}")
```

Comment:
Make sure to update the docs to reflect this. I'm pretty sure they say to search for "Error producing event"

Reply:
Good catch, thanks! Added a comment for now.